src/EnergyPlus/FanCoilUnits.cc

```diff
@@ -3096,15 +3096,19 @@ namespace FanCoilUnits {
                                                    PLR,
                                                    CompressorOnFlag);
                     }
-                }
-                Calc4PipeFanCoil(state, FanCoilNum, ControlledZoneNum, FirstHVACIteration, QUnitOut);
+                } else if ((CoolingLoad && QUnitOutMax > QZnReq && QZnReq < 0.0) || (HeatingLoad && QUnitOutMax < QZnReq && QZnReq > 0.0)) {
```

Comment:
Seems like a straightforward check for calculating the desired operating PLR for the fan coil.

Reply:
I had to check for condition where the load is greater than the available capacity and set PLR to 1.0. Previously Calc4PipeFanCoil() was run with out the PLR argument and you do have to worry when load exceed the capacity because the default value of PLR is 1.